### DISCUSSION TITLE
Cannot get anything to show on the CRT/VIO

### DISCUSSION THREAD

**Discussion by laffineuse** [thehighnibble/thehighnibble.github.io#109] [IMSAI8080esp replica]
I installed the latest June 2023 release yet I cannot get anything to show on the CRT, I keep getting the message "NO VIO".
I use the CP/M 2.2 startup configuration: 0000 1001 0101 0000
What am I doing wrong?  I checked the MicroSD card and I do see the VIO ROM in the ROM folder.

**Comment by GrantMeStrength** [comment]
There are other settings you might need to make in the config panel?

**Comment by donfroula** [comment]
You need to boot using Memory Map 2, which supports the VIO board.

I use "0000 1010 0101 1000"

Bit 11  - NVS_BANK_ROM = 1
Bits  8-10 - NVS_MEMORY_MAP = 2
Bit 7 - NVS_UNLIMITED = 1
Bit 4 - NVS_Z80 = 1
Bit 3 - NVS_IF_STA = 1

**Reply by laffineuse** [reply]
Thank you
